base/markdown/parse/util.jl

```diff
@@ -93,7 +93,7 @@ function startswith{T<:String}(stream::IO, ss::Vector{T}; kws...)
 end
 
 function startswith(stream::IO, r::Regex; eat = true, padding = false)
-  @assert beginswith(r.pattern, "^")
+  @assert Base.startswith(r.pattern, "^")
```

Comment:
Ideally the `Markdown.startswith ()` method should be renamed now?

@one-more-minute, isn't that your code?